quicktests/overlaying/tests/realistic/category_grid.js

```diff
@@ -65,7 +41,9 @@ function run(svg, data, Plottable) {
   plot.addDataset(new Plottable.Dataset(data));
   plot.x(function(d){ return d.hospital; }, xScale)
       .y(function(d) { return d.hour; }, yScale)
-  .attr("fill", function(d) { return d.percent; }, cs);
+      .label(function(d) { return d.percent.toString() + "%";  })
```

Comment:
The `toString()` is theoretically extraneous here, since adding `"%"` will auto-cast.